### PR TITLE
docs: Spelling corrections

### PR DESCRIPTION
**Pull request by mcauser**
paramter -> parameter
send a receive -> send and receive
respones -> response
succeses -> success
repsonse -> response
particualr -> particular
constructore -> constructor

### CODE REVIEW

File: cc3200/hal/prcm.c [hunk 1]
```diff
@@ -1015,7 +1015,7 @@ void PRCMSRAMRetentionDisable(unsigned long ulSramColSel, unsigned long ulFlags)
 //!
 //! \param ulHIBWakupSrc is logical OR of valid HIB wakeup sources.
 //!
-//! This function enables individual HIB wakeup source(s). The paramter
+//! This function enables individual HIB wakeup source(s). The parameter
```
Comment (dpgeorge) [inline cc3200/hal/prcm.c]: Please don't change this file.  It's from an upstream source.  You can try getting TI to change it if you really want :)

File: drivers/nrf24l01/nrf24l01test.py [hunk 2]
```diff
@@ -57,7 +57,7 @@ def master():
         # delay then loop
         pyb.delay(250)
 
-    print('master finished sending; succeses=%d, failures=%d' % (num_successes, num_failures))
+    print('master finished sending; success=%d, failures=%d' % (num_successes, num_failures))
```
Comment (dpgeorge) [inline drivers/nrf24l01/nrf24l01test.py]: This should be "successes"

**Review by dpgeorge — COMMENTED** [review]
Thanks, but please split it into 2 commits: one for the docs files and another for the drivers files.

**Comment by mcauser** [comment]
Bump

**Comment by dpgeorge** [comment]
Sorry, I didn't notice that you responded to my comments.  Now merged, see c5310ee5b545316d7b8770343908c19f1ff1cd0d.

**Pull request closed without merging**